src/modules/dashboard/components/AwardAndSmiteDialogForm/AwardAndSmiteDialogForm.tsx

```diff
@@ -283,12 +257,12 @@ const SmiteDialogForm = ({
         <div className={styles.singleUserContainer}>
           <SingleUserPicker
```

Comment:
As I commented in Olga's #2693 this also needs the first user in the colony, pre-selected in the user picker _(same way we do in the Permissions modal)_

Currently it is not:
![Screenshot from 2021-11-02 16-44-56](https://user-images.githubusercontent.com/1193222/139869973-8ec0a069-339b-484a-8a01-f208ed538fba.png)

Reply:
Once #2830 is merged I'll make sure the changes are compatible with the award dialog